### DISCUSSION TITLE
is the Favicon broken?

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [rupali-codes/LinksHub#2265] [General]
is it just me or is the Favicon for the site not working? not sure if it's an issue.

![icon](https://github.com/rupali-codes/LinksHub/assets/25883220/78b07fbf-c4b6-4fda-afeb-cffe5b71d822)

I test it in multiple browsers

**Comment by Anmol-Baranwal** [comment]
It seems to be visible in chrome.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/8cbca1fe-f1d5-4d82-a6fa-43cba36534c6)

Which browser are you using @JuanPabloDiaz

**Reply by JuanPabloDiaz** [reply]
Edge, GC, Brave
